opensir/models/model.py

```diff
@@ -57,18 +56,31 @@ def _model(self):
     def set_params(self, p, initial_conds):
         """ Set model parameters.
         Args:
-            p (list): parameters of the model. The parameters units are 1/day,
+            p (dict or list): parameters of the model. The parameters units are 1/day,
                       and should be >= 0.
```

Comment:
Is this indentation ok? It does not seem to be consistent with `initial_conds`

Reply:
Then initial_conds should change, as a tabbed second line is required to display well sphinx docs